### ISSUE TITLE
Statistical concepts

### ISSUE THREAD

**Issue by chfor183** [chfor183/data_science_articles#6]
## TL;DR

Yes !

## Key Takeaways
- 1
- 2

## Useful Code Snippets
```
function test() {
  console.log("notice the blank line before this function?");
}
```

## Articles/Ressources

https://medium.com/diogo-menezes-borges/introduction-to-statistics-for-data-science-6c246ed2468d
https://medium.com/diogo-menezes-borges/introduction-to-statistics-for-data-science-3087b80eb1c6
https://medium.com/diogo-menezes-borges/introduction-to-statistics-for-data-science-7bf596237ac6
https://medium.com/diogo-menezes-borges/introduction-to-statistics-for-data-science-16a188a400ca
https://medium.com/diogo-menezes-borges/introduction-to-statistics-for-data-science-a67a3199dcd4
https://towardsdatascience.com/statistical-learning-for-data-science-b61b263c1196
https://medium.com/@rathi.ankit/probability-statistics-for-data-science-series-83b94353ca48
https://towardsdatascience.com/type-i-and-type-ii-errors-of-hypothesis-tests-understand-with-graphs-43079fdd936a
https://towardsdatascience.com/the-5-basic-statistics-concepts-data-scientists-need-to-know-2c96740377ae
https://towardsdatascience.com/p-values-explained-by-data-scientist-f40a746cfc8
https://towardsdatascience.com/everything-you-need-to-know-about-hypothesis-testing-part-i-4de9abebbc8a
https://towardsdatascience.com/statistical-tests-when-to-use-which-704557554740
https://towardsdatascience.com/demystifying-statistical-analysis-1-a-handy-cheat-sheet-b6229bf992cf
https://towardsdatascience.com/the-ultimate-guide-to-a-b-testing-part-2-data-distributions-5ed429abbce
https://towardsdatascience.com/a-brief-introduction-of-some-common-statistical-tests-5f2963014641
https://towardsdatascience.com/how-to-assess-statistical-significance-in-your-data-with-permutation-tests-8bb925b2113d
https://medium.com/swlh/daily-data-science-101-bootstrap-c0895dc10535
https://towardsdatascience.com/the-central-limit-theorem-and-its-implications-4a7adac9d6de
https://medium.com/towards-artificial-intelligence/3-ways-linear-models-can-lead-to-erroneous-conclusions-842637fe122b
https://medium.com/cracking-the-data-science-interview/the-10-statistical-techniques-data-scientists-need-to-master-1ef6dbd531f7
